app/models/user.rb

```diff
@@ -1,19 +1,24 @@
 class User < ApplicationRecord
-  has_many :posts, foreign_key: :author_id, dependent: :destroy
-  has_many :comments, foreign_key: :author_id, dependent: :destroy
-  has_many :likes, foreign_key: :author_id, dependent: :destroy
+  devise :database_authenticatable, :registerable, :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Kindly add `:confirmable` to the user model and make sure the user can confirm their accounts before logging in, as per [this](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_devise.md#:~:text=Ask%20for%20confirmation%20of%20email.) requirement.

Reply:
Okay, Thank you.